opt/Makefile

```diff
@@ -134,8 +134,10 @@ ifneq ($(NO_LFS),1)
 endif
 	$(SHOW)set -e ;\
 	cd $(ROOT)/test ;\
+	python3 -m RLTest $(TEST_ARGS) --test basic_tests.py --module $(INSTALL_DIR)/redisai.so ;\
 	python3 -m RLTest $(TEST_ARGS) --test basic_tests.py --module $(INSTALL_DIR)/redisai.so --use-slaves ;\
-	python3 -m RLTest $(TEST_ARGS) --test  double-panda.py --module $(INSTALL_DIR)/redisai.so
```

Comment:
This was failing on macOS, so this line actually waits for the macOS CI to be enabled. If we're convinced that everything is in order there, we can remove the rest of the test artifacts. Otherwise, it may be worthwhile to just comment it out until CI is ready.